pkg/test/props/Directory.Build.props

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <ImportDirectoryBuildTargets>false</ImportDirectoryBuildTargets>
```

Comment:
What issue does this solve?

Reply:
Package testing was busted because it was pulling in the Directory.Build.* from the root folder of the repo and setting things like the TargetFramework property.  This ensures that it doesn't do that.

Reply:
right, this will basically turn off the import of the Directory.Build.targets. We do this in the props since that is imported before this, and we can't do this on the actual projects since the import of the files happens before the project. Alternatively, we could have provided an empty Directory.Build.targets as well.